modules/cron/lib/Cron.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Handles interactions with SSP's cron system/hooks.
+ */
+class sspmod_cron_Cron
```

Comment:
Since this is a new class (and the only one in this module), I would actually prefer it to use the PSR-4-compliant naming convention. That would mean naming the class `Cron` inside the `SimpleSAML\Module\cron` namespace.